{"name": "test_login_invalid_credentials", "status": "passed", "description": "Проверка:неправильные логин или пароль возвращают ошибку \"Учетная запись не найдена\", статус 404", "start": 1722980482259, "stop": 1722980482452, "uuid": "535fdf87-421a-4320-aed7-1d2653d643ca", "historyId": "ec706951e0c7470079769d6d675d4cda", "testCaseId": "ec706951e0c7470079769d6d675d4cda", "fullName": "tests.test_login_courier.TestLoginCourier#test_login_invalid_credentials", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login_courier"}, {"name": "subSuite", "value": "TestLoginCourier"}, {"name": "host", "value": "LAPTOP-74V5TFPG"}, {"name": "thread", "value": "17284-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_courier"}]}